### ISSUE TITLE
Issues preventing the Application from Launching

### ISSUE THREAD

**Issue by CarJem** [CarJem/ManiacEditor-GenerationsEdition#16]

**Comment by DraconicChaos901** [comment]
Having this issue as well. I'll try using some of the older versions to see if it's a version-specific issue.
Update: Okay, never mind. The issue is that the latest release link is still pointing to a version that has this issue... Or at least that's what I had an issue with.

**Comment by CarJem** [comment]
Yeah, I'm aware there's something odd going on here.... but without crash logs I can't address the issue.

**Comment by MDTravisYT** [comment]
This appears to be an issue when extracting the zip in the same folder where the Mania exe's at. It works fine if placed in its own folder.
